src/features/Movies/MovieList/moviesSlice.js

```diff
@@ -17,6 +18,9 @@ const moviesSlice = createSlice({
       state.loading = "success";
       state.movies = movies;
     },
+    updateSearchTerm: (state, action) => {
+      state.searchTerm = action.payload;
```

Comment:
Maybe it's better like this: updateSearchTerm: (state, { payload: action}) => {
      state.searchTerm = action;`
}